README.md

```diff
@@ -1,2 +1,23 @@
-# lazy-action-setup
+# Lazy Action Setup
+
 Sets environment variables that will be used by other Lazy Actions
+
+## Environment Variables
+
+Following environment variables are set by default in addition to `gitversion`
+
+```text
+SONAR_TOKEN
+SONAR_ORG
+AWS_ROLE_ARN
+AWS_ROLE_ARN
+IMAGE_VERSION
+```
+
+## Output Variables
+
+Following output variables are set by default in addition to `gitversion`
+
+```text
+image-version
```

Comment:
Can you add tables here just like you added in `lazy-action-dotnet`? I liked them.